kubespawner/spawner.py

```diff
@@ -70,7 +70,7 @@ class EventReflector(NamespacedResourceReflector):
     def events(self):
         return sorted(
             self.resources.values(),
-            key=lambda x: x.last_timestamp,
+            key=lambda x: x.last_timestamp and x.last_timestamp.timestamp() or 0.,
```

Comment:
```suggestion
            key=lambda x: x.last_timestamp if x.last_timestamp is not None else 0.,
```

I'd use this format as that is the official ternary operator in Python

Reply:
We should use
```python
key=lambda x: x.last_timestamp.timestamp() if x.last_timestamp is not None else 0.,
```
instead of
```python
key=lambda x: x.last_timestamp if x.last_timestamp is not None else 0.,
```